src/ecrc-api/src/test/java/ca/bc/gov/open/ecrc/service/EcrcWebMethodsServiceImplTest.java

```diff
@@ -115,10 +115,12 @@ public void testWebMethodsCallUnknown() throws JsonProcessingException {
         Assertions.assertEquals(jsonUnknownResp, res.getBody());
     }
 
-    @DisplayName("Failure - webMethods call bad json")
+	@DisplayName("Failure - webMethods call bad json")
```

Comment:
Very minor: looks like the spacing got misaligned here compared to the surrounding code. Looks good to me otherwise

Reply:
Thats because of using both IntelliJ and Eclipse. The formatting is out of sync and it needs to be fixed eventually.